test/functional/example_test.py

```diff
@@ -78,7 +78,10 @@ def set_test_params(self):
         """Override test parameters for your individual test.
 
         This method must be overridden and num_nodes must be explicitly set."""
-        self.setup_clean_chain = True
+        # By default the test loads a pre-mined chain of 200 blocks from cache.
+        # Set skip_chain_setup to True to skip this and start from the Genesis
```

Comment:
"True to not do something" seems like a naming regression. True to set up a clean chain is clearer.